### PR TITLE
feat: Rename other tab to write

### PR DESCRIPTION
**Pull request by laurenzlong**
Users have found the "Other" tab to be confusingly named. 

Screenshot 2024-12-06 at 1 27 55 PM (2)

### CODE REVIEW

File: src/components/Configure/nav/ObjectManagementNav/v2/Tabs/index.tsx [hunk 1]
```diff
@@ -111,8 +111,7 @@ export function VerticalTabs({
           <OtherTab
             completed={otherNavObject.completed}
             pending={objectConfigurationsState?.other?.write?.isWriteModified || false}
-             // if read tab exists, display 'other' else 'write' when write tab is the only tab
-            displayName={readNavObjects.length ? 'Other' : 'Write'}
+            displayName="Write"
```
Comment (laurenzlong) [inline src/components/Configure/nav/ObjectManagementNav/v2/Tabs/index.tsx:114]: This is the only substantive change, everything else is just renaming variables.

File: src/components/Configure/nav/ObjectManagementNav/constant.tsx [hunk 2]
```diff
@@ -1,2 +1,4 @@
-export const OTHER_CONST = 'other';
+// WRITE_CONST is used for the "Write" tab, its value is "other" for legacy reasons.
+export const WRITE_CONST = 'other';
```
Comment (dionlow) [inline src/components/Configure/nav/ObjectManagementNav/constant.tsx:2]: you can probably rename 'other' to 'write' here. I don't see 'other' hard coded anywhere here. just references to the enum.

**Comment by dionlow** [comment]
This branch doesn't build. There's a refactor naming issue.

**Review by dionlow — APPROVED** [review]
looks good

**Pull request merged**